src/main/scala/org/zalando/spearheads/innkeeper/dao/RoutesPostgresRepo.scala

```diff
@@ -73,13 +73,8 @@ class RoutesPostgresRepo @Inject() (
   override def selectAll: DatabasePublisher[RouteRow] = {
     logger.debug("select all")
 
-    val q = for {
-      routeRow <- routesTable
-      if routeRow.deletedAt.isEmpty
-    } yield routeRow
-
     db.stream {
-      q.result
+      selectAllQuery.result
```

Comment:
wow... This significantly changes the behaviour of the method (previously: return all non-deleted routes, now: return all routes). I hope, all code, that relied on the previous behaviour has been updated?